code/modules/jobs/job_types/ncr.dm

```diff
@@ -367,7 +367,6 @@ Veteran Ranger
 	suit = 			/obj/item/clothing/suit/armor/f13/rangercombat
 	head = 			/obj/item/clothing/head/helmet/f13/ncr/rangercombat
 	gloves =		/obj/item/clothing/gloves/fingerless
-	suit_store = 	/obj/item/gun/ballistic/shotgun/automatic/antimateriel
```

Comment:
1) No.
2) Fuck no. 
3) Detail this in the PR.
4) You're leaving VRs without a primary weapon.

Reply:
Also make 45LC craftable in the lathe if you're giving 4570s to anyone but the VR.